Content.Server/Devour/Components/DevourerComponent.cs

```diff
@@ -0,0 +1,83 @@
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Devour;
+using Content.Shared.Mobs;
+using Content.Shared.Whitelist;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.Devour.Components
+{
+    [RegisterComponent]
+    public sealed class DevourerComponent : Component
```

Comment:
use access attribute, also move to shared

Reply:
Please look at my other comment, https://github.com/space-wizards/space-station-14/pull/16345#discussion_r1192956936 about "shared" part.